c/src/ml-api-service-training-offloading.c

```diff
@@ -240,6 +245,12 @@ _training_offloading_conf_parse_json (ml_service_s * mls, JsonObject * object)
   training_node = json_object_get_member (object, "training");
   training_obj = json_node_get_object (training_node);
 
+  receive_time_limit =
+      json_object_get_int_member (training_obj, "receive-time(s)-limit");
```

Comment:
suggestion : json key for time-limit and prevent warning message.
```
when init training_s struct: training_s->time_limit =DEFAULT_TIME_LIMIT;
...
if (json_object_has_member (training_obj, "time-limit")) {
  training_s->time_limit = json_object_get_int_member (training_obj, "time-limit");
}
```

Reply:
How about specifying what is ```time-limit```?
receive-time(sec)-limit is not okay?